### ISSUE TITLE
Line breaks are inserted inside a {@link} tag

### ISSUE THREAD

**Issue by sluukkonen** [hosseinmd/prettier-plugin-jsdoc#107]
A comment like

 ```typescript
/**
 * Calculate the 
 * {@link https://en.wikipedia.org/wiki/Complement_(set_theory)#Relative_complement difference}
 * between two sets.
 */
export function difference<T>(first: Set<T>): (second: Set<T>) => Set<T>
```

is formatted as 

```typescript
/**
 * Calculate the {@link
 * https://en.wikipedia.org/wiki/Complement_(set_theory)#Relative_complement
 * difference} between two sets.
 */
export function difference<T>(first: Set<T>): (second: Set<T>) => Set<T>
```

which breaks the rendering in at least Webstorm.

![kuva](https://user-images.githubusercontent.com/39655/115301953-8d0d5a80-a16a-11eb-98a5-c0661cf37c78.png)

Configuration:

```
{
  "semi": false,
  "singleQuote": true,
  "plugins": ["prettier-plugin-jsdoc"],
  "tsdoc": true
}
```

**Comment by danielpza** [comment]
for reference this are the docs https://jsdoc.app/tags-inline-link.html

**Comment by jaydenseric** [comment]
Some other similar issues…

Markdown links in descriptions are wrapped strangely:

<img width="659" alt="Screen Shot 2021-04-23 at 9 27 06 am" src="https://user-images.githubusercontent.com/1754873/115797010-3fb51580-a416-11eb-840d-d84d107b3d49.png">

Sometimes it forces a blank space after the opening `[`:

<img width="661" alt="Screen Shot 2021-04-23 at 9 28 58 am" src="https://user-images.githubusercontent.com/1754873/115797088-6a06d300-a416-11eb-92ee-98da9c4b2a00.png">

This also happens for JSDoc links:

<img width="664" alt="Screen Shot 2021-04-23 at 9 26 35 am" src="https://user-images.githubusercontent.com/1754873/115797113-79861c00-a416-11eb-8769-60a9fffab7a1.png">

**Comment by hosseinmd** [comment]
Fixed both in [chore(release): 0.3.20](https://github.com/hosseinmd/prettier-plugin-jsdoc/commit/176be1d874bb755411f9f0a7a6fa8d54d86699ad)
